src/test/java/teammates/test/driver/HtmlHelper.java

```diff
@@ -22,53 +22,45 @@
 public class HtmlHelper {
 
     /**
-     * Verifies that two HTML files are logically 
-     * equivalent e.g., ignores differences in whitespace and attribute order.
+     * Verifies that two HTML files are logically equivalent, e.g. ignores
+     * differences in whitespace and attribute order. If the assertion fails,
+     * <code>AssertionError</code> will be thrown and the difference can then be traced.
+     * @param expectedString the expected string for comparison
+     * @param actualString the actual string for comparison
+     * @param isPart if true, ignores top-level HTML tags, i.e <code>&lt;html&gt;</code>,
```

Comment:
It's a known issue that I know how to fix already, and will fix by this or latest next milestone. Should I still add it?